tools/setup

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+ROOT=$(dirname $0)/..
+
+# 1. Install nvm
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | sh 
```

Comment:
check return codes on these and give meaningful feedback.  Something like this:

`myfoocmd || (echo 'uhoh'; exit)`

Also extract the version number for nvm (0.34.0) into a variable so it can be adjusted later.